docs/dagster-university/pages/dagster-essentials/lesson-4/coding-practice-taxi-zones-asset.md

```diff
@@ -30,7 +30,7 @@ The asset you built should look similar to the following code. Click **View answ
     deps=["taxi_zones_file"]
 )
 def taxi_zones() -> None:
-    sql_query = f"""
+    query = f"""
```

Comment:
Standardizing around `query` as the variable name since that is what the dagster university repo uses in functions